docs/tutorials/prefill_profiler.rst

```diff
@@ -40,7 +41,8 @@ To profile a custom range of prompt lengths, use the flag ``--prefill-lengths``
     --model "meta-llama/Meta-Llama-3-8B-Instruct" \
     --max-num-completed-requests 1 \
     --timeout 600 \
-    --num-concurrent-requests 1 \
+    --num-ray-clients 1 \
```

Comment:
let's put some good values here?

Reply:
Resolved. Hardcoded these values inside prefill profiler itself as we are always going to send only 1 request for a given prompt length.